scripts/forward_logfiles.sh

```diff
@@ -0,0 +1,171 @@
+        mkdir -p /var/vcap/sys/rsyslog/buffered
+        chown syslog:adm /var/vcap/sys/rsyslog/buffered
+
+	case "${SCF_LOG_PROTOCOL}" in
+	    udp)
+		SCF_LOG_PREFIX=
+		;;
+	    tcp)
+		SCF_LOG_PREFIX=@
+		;;
+	    *)
+                echo "Rsyslog forwarder: Bad protocol ${SCF_LOG_PROTOCOL}, could not create ${MAIN_CONFIG} in ${RSYSLOG_CONF_DIR}" >> "${PB_OUT}"
+                exit 0
+		;;
+	esac
+
+        if ! sed 's@^\s*@@' >"${RSYSLOG_CONF_DIR}/${MAIN_CONFIG}" <<-EOF ; then
+                module(load="imfile" mode="polling")
+                \$template RFC5424Format,"<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% - - - %msg:::drop-last-lf%"
+                \$RepeatedMsgReduction on
+                \$MaxMessageSize 4k                             # default is 2k
+                \$WorkDirectory /var/vcap/sys/rsyslog/buffered  # where messages should be buffered on disk
+                \$ActionResumeRetryCount -1                     # Try until the server becomes available
+                \$ActionQueueType LinkedList                    # Allocate on-demand
+                \$ActionQueueFileName agg_backlog               # Spill to disk if queue is full
+                \$ActionQueueMaxDiskSpace 32m                   # Max size for disk queue
+                \$ActionQueueLowWaterMark 2000                  # Num messages. Assuming avg size of 512B, this is 1MiB.
+                \$ActionQueueHighWaterMark 8000                 # Num messages. Assuming avg size of 512B, this is 4MiB.
+                                                                # (If this is reached, messages will spill to disk until the low watermark is reached).
+                \$ActionQueueTimeoutEnqueue 0                   # Discard messages if the queue + disk is full
+                \$ActionQueueSaveOnShutdown on                  # Save in-memory data to disk if rsyslog shuts down
+                :app-name, startswith, "vcap." @${SCF_LOG_PREFIX}${SCF_LOG_HOST}:${SCF_LOG_PORT};RFC5424Format
+                :app-name, startswith, "vcap." ~                # Stop writing SCF message logs to /var/log
+	EOF
+                echo "Rsyslog forwarder: Could not create ${MAIN_CONFIG} in ${RSYSLOG_CONF_DIR}" >> "${PB_OUT}"
+                exit 0
+        fi
+
+        if [[ ! -f "${RSYSLOG_CONF_DIR}/${MAIN_CONFIG}" ]]; then
+                echo "Rsyslog forwarder: File ${MAIN_CONFIG} not found in ${RSYSLOG_CONF_DIR}" >> "${PB_OUT}"
+                exit 0
+        fi
+}
+
+# check if more logs to be monitored by rsyslog have come into existence since the last run
+function searchTargetDir {
+        local dir file filesAdded=1
+        local args=( "$1" )
+        for dir in ${IGNORE_DIR} ; do
+                args=( "${args[@]}" '(' -name "${dir}" -a -prune ')' -o )
+        done
+        args=( "${args[@]}" '(' -name '*.log' -a -type f -a -print0 ')' )
+        while IFS= read -r -d '' file ; do
+                TARGET_BASENAME="$(basename "$1" .log)"
```

Comment:
This needs the same fix as in SCF. ... Actually, just copying the fixed script should be good enough, it looks to be identical.